LiskHQ/lisk-sdk
# Use nodejs crypto library for checksum verification - Closes #3872

### What was the problem?
`shasum` was incompatible with `linux` when the machine had bare minimum installation.
### How did I fix it?
Using Nodejs `crypto` library to verify the checksum to support all OS.
### How to test it?
- Use a bare Linux machine or Install Docker on your machine and run docker run -it ubuntu:18.04 /bin/bash
- Create lisk user and install commander 2.2.0
- Extract and install Lisk core ./lisk core:install --network=testnet testnet-01
- The installation should succeed

### Review checklist

* The PR resolves #3872 
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
